climate_categories/data/CRF2013.yaml

```diff
@@ -0,0 +1,6073 @@
+    - 5B2a
+  5.B.2.b:
+    title: Other (Please Specify)
+    alternative_codes:
+    - 5 B 2 b
+    - 5B2b
+  5.C.1.a:
+    title: Biogenic
+    alternative_codes:
+    - 5 C 1 a
+    - 5C1a
+    children:
+    - - 5.C.1.a.i
+      - 5.C.1.a.ii
+  5.C.1.a.i:
+    title: Municipal Solid Waste
+    alternative_codes:
+    - 5 C 1 a i
+    - 5C1ai
+  5.C.1.a.ii:
+    title: Other (please specify)
+    alternative_codes:
+    - 5 C 1 a ii
+    - 5C1aii
+    children:
+    - - 5.C.1.a.ii.1
+      - 5.C.1.a.ii.2
+      - 5.C.1.a.ii.3
+      - 5.C.1.a.ii.4
+      - 5.C.1.a.ii.5
+  5.C.1.a.ii.1:
+    title: Industrial Solid Waste
+    alternative_codes:
+    - 5 C 1 a ii 1
+    - 5C1aii1
+  5.C.1.a.ii.2:
+    title: Hazardous Waste
+    alternative_codes:
+    - 5 C 1 a ii 2
+    - 5C1aii2
+  5.C.1.a.ii.3:
+    title: Clinical Waste
+    alternative_codes:
+    - 5 C 1 a ii 3
+    - 5C1aii3
+  5.C.1.a.ii.4:
+    title: Sewage Sludge
+    alternative_codes:
+    - 5 C 1 a ii 4
+    - 5C1aii4
+  5.C.1.a.ii.5:
+    title: Other (Please Specify)
+    alternative_codes:
+    - 5 C 1 a ii 5
+    - 5C1aii5
+  5.C.1.b:
+    title: Non-Biogenic
+    alternative_codes:
+    - 5 C 1 b
+    - 5C1b
+    children:
+    - - 5.C.1.b.i
+      - 5.C.1.b.ii
+  5.C.1.b.i:
+    title: Municipal Solid Waste
+    alternative_codes:
+    - 5 C 1 b i
+    - 5C1bi
+  5.C.1.b.ii:
+    title: Other (please specify)
+    alternative_codes:
+    - 5 C 1 b ii
+    - 5C1bii
+    children:
+    - - 5.C.1.b.ii.1
+      - 5.C.1.b.ii.2
+      - 5.C.1.b.ii.3
+      - 5.C.1.b.ii.4
+      - 5.C.1.b.ii.5
+  5.C.1.b.ii.1:
+    title: Industrial Solid Waste
+    alternative_codes:
+    - 5 C 1 b ii 1
+    - 5C1bii1
+  5.C.1.b.ii.2:
+    title: Hazardous Waste
+    alternative_codes:
+    - 5 C 1 b ii 2
+    - 5C1bii2
+  5.C.1.b.ii.3:
+    title: Clinical Waste
+    alternative_codes:
+    - 5 C 1 b ii 3
+    - 5C1bii3
+  5.C.1.b.ii.4:
+    title: Sewage Sludge
+    alternative_codes:
+    - 5 C 1 b ii 4
+    - 5C1bii4
+  5.C.1.b.ii.5:
+    title: Other (Please Specify)
+    alternative_codes:
+    - 5 C 1 b ii 5
+    - 5C1bii5
+  5.C.2.a:
+    title: Biogenic
+    alternative_codes:
+    - 5 C 2 a
+    - 5C2a
+    children:
+    - - 5.C.2.a.i
+      - 5.C.2.a.ii
+  5.C.2.a.i:
+    title: Municipal Solid Waste
+    alternative_codes:
+    - 5 C 2 a i
+    - 5C2ai
+  5.C.2.a.ii:
+    title: Other (please specify)
+    alternative_codes:
+    - 5 C 2 a ii
+    - 5C2aii
+  5.C.2.b:
+    title: Non-Biogenic
+    alternative_codes:
+    - 5 C 2 b
+    - 5C2b
+    children:
+    - - 5.C.2.b.i
+      - 5.C.2.b.ii
+  5.C.2.b.i:
+    title: Municipal Solid Waste
+    alternative_codes:
+    - 5 C 2 b i
+    - 5C2bi
+  5.C.2.b.ii:
+    title: Other (please specify)
+    alternative_codes:
+    - 5 C 2 b ii
+    - 5C2bii
+  '0':
+    title: Total National Emissions and Removals
+    children:
+    - - '1'
+      - '2'
+      - '3'
+      - '4'
+      - '5'
+  M.Memo.IndN2O:
+    title: 'Indirect N2O '
```

Comment:
There's an extraneous space at the end here, maybe `strip()` category titles in the genration?